lib/chai/core/assertions.js

```diff
@@ -1961,7 +1961,7 @@ module.exports = function (chai, _) {
   /**
    * ### .increase(function)
```

Comment:
Maybe the docs for this could use another name for the argument being passed other than `function`.